lib/active_shipping/delivery_date_estimate.rb

```diff
@@ -0,0 +1,19 @@
+module ActiveShipping
+  class DeliveryDateEstimate
```

Comment:
Does this API call also include rates?

I'd prefer it if this class inherits from `RateResponse`, or at least make it duck type compatible with RateResponse for the same reason I noted above.